backend/src/test/java/com/woowacourse/levellog/acceptance/TeamAcceptanceTest.java

```diff
@@ -183,6 +184,39 @@ void findTeam_notLogin() {
                         "interviewees", empty());
     }
 
+    /*
+     * Scenario: 팀 상태 조회하기
+     *   given: 팀이 등록되어 있다.
+     *   when: 로그인하지 않고 팀 상태 조회를 요청한다.
+     *   then: 200 Ok 상태 코드와 팀 상태를 응답받는다.
+     */
+    @Test
+    @DisplayName("팀 상태 조회하기")
+    void findStatus() {
+        // given
+        final RestAssuredResponse pepper = login("페퍼");
+        final Long rickId = login("릭").getMemberId();
+
+        final ParticipantIdsDto participants = new ParticipantIdsDto(List.of(rickId));
+        final TeamWriteDto teamDto = new TeamWriteDto("잠실 제이슨조", "트랙룸", 1, TEAM_START_TIME, participants);
+
+        final String id = post("/api/teams", pepper.getToken(), teamDto).getTeamId();
+
+        timeStandard.setInProgress(); // 인터뷰 시작
+
+        // when
+        final ValidatableResponse response = RestAssured.given(specification).log().all()
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .filter(document("team/find-status"))
+                .when()
+                .get("/api/teams/{id}/status", id)
+                .then().log().all();
+
+        // then
+        response.statusCode(HttpStatus.OK.value())
+                .body("status", equalTo(String.valueOf(TeamStatus.IN_PROGRESS)));
```

Comment:
```suggestion
                .body("status", equalTo(TeamStatus.IN_PROGRESS.name()));
```
Enum에서 제공하는 name()이나 재정의 가능한 toString()을 사용하는 건 어떨까유